spec/controllers/questions/consent_controller_spec.rb

```diff
@@ -239,8 +239,8 @@
           BODY
 
           expect(ClientMessagingService).to have_received(:send_system_text_message).with(
-            intake.client,
-            body.chomp,
+              client: intake.client,
+              body: body.chomp
```

Comment:
It's great that we have these unit tests; they effectively validate the use of `**args`.